activate.ps1

```diff
@@ -0,0 +1,57 @@
+#
+# This file must be used by invoking ". .\activate.ps1" from the command line.
+# You cannot run it directly.
+# To exit from the environment this creates, execute the 'deactivate' function.
+#
+
+function deactivate ([switch]$init) {
+
+    # reset old environment variables
+    if (Test-Path variable:_OLD_PATH) {
+        $env:PATH = $_OLD_PATH
+        Remove-Item variable:_OLD_PATH
+    }
+
+    if (test-path function:_old_prompt) {
+        Set-Item Function:prompt -Value $function:_old_prompt -ea ignore
+        remove-item function:_old_prompt
+    }
+
+    Remove-Item env:DOTNET_ROOT -ea ignore
```

Comment:
Does everyone have PowerShell 3.0 or later? `ignore` wasn't supported in earlier releases and 3.0 was only an add-on for Windows before Windows 8 and Windows Server 2012.

Reply:
We require PowerShell 5 or newer to build this repo, so I'm not too worried about this: https://github.com/aspnet/AspNetCore/blob/94a4293cce735b3756661ef5a475c4944f57d64d/build.ps1#L1

Also, with #7005, we're also going to require Windows 10.